enterprise/server/backends/userdb/userdb.go

```diff
@@ -164,15 +164,15 @@ func (d *UserDB) getAPIKey(tx *db.DB, apiKeyID string) (*tables.APIKey, error) {
 	return key, nil
 }
 
-func (d *UserDB) GetAPIKeys(ctx context.Context, groupID string, checkVisibility bool) ([]*tables.APIKey, error) {
+func (d *UserDB) GetAPIKeys(ctx context.Context, groupID string, authorizeRole bool) ([]*tables.APIKey, error) {
```

Comment:
can you add a comment here about when authorizeRole should be set / why?

Reply:
@tylerwilliams Split out https://github.com/buildbuddy-io/buildbuddy/pull/3357/files, PTAL

Reply:
Rebased this PR, should be good to go